src/plugins/intel_cpu/src/nodes/kernels/scaled_attn/executor_pa.cpp

```diff
@@ -2473,13 +2497,15 @@ struct MHAHelper {
                                                precision_of<DATA_TYPE>::value,
                                                alibi_slope);
                 }
-                if (score_output) {
-                    cvt_copy(score_output + h * rnd_up(cur_kv_len, 16),
-                             reinterpret_cast<DATA_TYPE*>(score),
-                             1,
-                             cur_kv_len,
-                             0,
-                             0);
+                if (score_output && m >= q_start_idx_score) {
+                    cvt_add(score_output + h * score_info_ptr->kv_len_aligned * score_info_ptr->score_buf_num,
```

Comment:
The in-place add could be simplified for better understanding
```
auto score_block_ptr = score_output + h * score_info_ptr->kv_len_aligned * score_info_ptr->score_buf_num
cvt_add(score_block_ptr, score_block_ptr ....)
```

Reply:
Done.